tcms/core/utils/__init__.py

```diff
@@ -60,7 +60,7 @@ def clean_request(request, keys=None):
     """
```

Comment:
`clean_request()` is only used inside `testruns/views.py:open_run_get_case_runs()` which in turn is only used inside `testruns/views.py:get()`, which renders a particular test run. E.g. `/run/4/`. Several lines of this and calling functions are not covered via tests!

Now as far as I understand we could pass query string (GET) parameters when viewing a TR via its URL. For example `http://127.0.0.1:8000/run/4/?order_by=case__summary` to order cases by their summary, not by sortkey/pk.

From what I can tell nowhere in Nitrate we use the functionality to load a TR so that it's cases are sorted differently.  The url pattern for `views.get()` is also not named, meaning it's not used for reverse or somewhere else. 

From what I can tell this entire function can be removed. Also the section in `open_run_get_case_runs()` where we sort the existing `tcrs` can be removed. Objections ?
`